python/pip_install/entrypoint.bzl

```diff
@@ -0,0 +1,60 @@
+# Copyright 2023 The Bazel Authors. All rights reserved.
```

Comment:
Do you know if this is private?  Would a customer reuse this code inside their project?  If not, please put it into a private folder.

Reply:
Added a note why this can't be private.